client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -811,11 +811,9 @@ page-header, .page-header--new {
 // Forms
 .form-control {
   border-radius: @redash-input-radius;
-  color: #9E9E9E;
 
   &:focus {
     box-shadow: none !important;
-    color: #111;
```

Comment:
Why remove this?

Reply:
I was about to comment this line to ask :joy:. Ant has no color styling for focused inputs, I could either create a selector with a different color for antd or remove this to have the same behavior.

Reply:
Oh. Well, if we switch to Ant's darker color everywhere, then we don't need this. Otherwise better to keep as without it the text is harder to read.

Reply:
> Oh. Well, if we switch to Ant's darker color everywhere, then we don't need this. Otherwise better to keep as without it the text is harder to read.
D'accord :+1:. Do you have a reason to update to Ant's darker color later (wait for discussion or ath) or should I update it here?

Reply:
The reason I suggested to do it in a follow up before was to be able to merge this already. But considering you did the heavy lifting already, we can do this. Or we can merge this and still do the change to darker color in a follow up.

I will let @ranbena decide :)